cmd/ci-operator-configresolver/main.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/ghodss/yaml"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/interrupts"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/config"
+)
+
+const (
+	orgQuery     = "org"
```

Comment:
The main thing we gain is that now we are using a named constant so we are less prone to typos if we want to use any of these values elsewhere in this file. Typos should be caught by tests, but it is nice to have the IDE and `go vet` immediately identify that something is wrong. We also only have to change the value in one location instead of 2 if we decide the change a value.

Reply:
>  if we want to use any of these values elsewhere in this file

Do we?